examples/systemcalls/systemcalls.c

```diff
@@ -92,6 +119,39 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *   The rest of the behaviour is same as do_exec()
  *
 */
+    int status;
+    pid_t pid;
+    pid = fork();
```

Comment:
@Suhas-Reddy-S think about how you could avoid code duplication with the function above.